### ISSUE TITLE
RTL.conf hashed password

### ISSUE THREAD

**Issue by seth586** [ShahanaFarooqui/RTL#125]
**Is your feature request related to a problem? Please describe.**
Passwords are currently stored in plaintext in RTL.conf

**Describe the solution you'd like**
Allow specify a password hash in RTL.conf, at login RTL hashes typed password and compares with stored hash. This way if RTL filesystem is compromised the password is not exposed.

**Describe alternatives you've considered**
Perhaps this is unnecessary, since an attacker cold do worse things with a compromised filesystem...?

**Comment by tehelsper** [comment]
I agree. I added this feature request which, if implemented, would solve this issue, but just hashing the password would be a huge improvement.

https://github.com/ShahanaFarooqui/RTL/issues/115

I tried quickly adding it myself, but didn't have a lot of success. I'd need more understanding of how the config file is parsed and how it get used throughout RTL. My attempt added a new nodeAuthType for CUSTOM_HASH that performs a sha256 hash of the entered pw and compares it to a hash saved in rtlHashedPass in the config.
